source/constructs/lib/agent/delete-agent-resources/delete_agent_resources.py

```diff
@@ -111,13 +114,13 @@ def cleanup_crawlers():
             if database_name.startswith(f"{solution_name}-s3-"):
                 unstructured_database_name = database_name.replace(f"{solution_name}-s3-",f"{solution_name}-unstructured-",1)
                 remove_database(unstructured_database_name)
-            if len(response_crawler['Crawler']['Targets']['JdbcTargets']) == 1:
+            if len(response_crawler['Crawler']['Targets']['JdbcTargets']) > 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)